contracts/AssetPool.sol

```diff
@@ -131,6 +131,7 @@ contract AssetPool is Ownable, IAssetPool {
     function receiveApproval(
         address from,
         uint256 amount,
+        uint256 minAmountToMint,
```

Comment:
If we want to add any more validations, we should do it via calldata, not adding one more parameter.

Reply:
Right, done in d3a8f12